src/bun.js/api/filesystem_router.zig

```diff
@@ -88,11 +89,11 @@ pub const FileSystemRouter = struct {
             if (!(root_dir_path_.len == 0 or strings.eqlComptime(root_dir_path_.slice(), "."))) {
                 // resolve relative path if needed
                 const path = root_dir_path_.slice();
-                if (bun.path.Platform.isAbsolute(.auto, path)) {
+                if (path_handler.Platform.auto.isAbsolute(path)) {
```

Comment:
☝️ Consistent access syntax (as follows other places in the code) for `path_handler.Platform.auto.isAbsolute(path)` instead of passing the platform enum into the `isAbsolute(.auto,...` method.